packages/fmg-nitro-adjudicator/test/nitro-adjudicator.test.ts

```diff
@@ -222,16 +221,53 @@ describe('nitroAdjudicator', () => {
       });
 
       it('fires a deposited event', async () => {
-        const filter = nitro.filters.Deposited(null, null, null);
-        const { emitterWitness, eventPromise } = expectEvent(nitro, filter);
         const channelID = getChannelID(channel);
-        await depositTo(channelID);
-        const event = await eventPromise;
+        expectEvent(await (await depositTo(channelID)).wait(), 'Deposited', {
+          destination: channelID,
+          amountDeposited: bigNumberify(DEPOSIT_AMOUNT),
+        });
+      });
 
-        expect(emitterWitness).toBeCalled();
+      it('reverts when holdings is less than or equal to the expected level', async () => {
+        const channelID = getChannelID(channel);
+        await expectRevert(
+          () => depositTo(channelID, 10, 5),
+          'Deposit: holdings[destination] is less than expected',
+        );
+      });
 
-        expect(event.args.destination).toEqual(channelID);
-        expect(event.args.amountDeposited).toEqual(bigNumberify(DEPOSIT_AMOUNT));
+      it('refunds the entire deposit when holdings is greater than the expected held plus the amount sent', async () => {
+        const channelID = getChannelID(channel);
+        {
+          await expectEvent(await depositTo(channelID, DEPOSIT_AMOUNT, 0), 'Deposited', {
+            destination: channelID,
+            amountDeposited: bigNumberify(DEPOSIT_AMOUNT),
+          });
+        }
+
+        {
+          await expectEvent(await depositTo(channelID, DEPOSIT_AMOUNT, 0), 'Deposited', {
```

Comment:
It might be nice to add a check that the funds were actually refunded in this test. Just to catch any cases where what we `transfer` doesn't line up with the `deposited` event for some reason.